js/controller.js

```diff
@@ -806,6 +813,13 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
 
     onVcPlay: function(event, source) {
       this.state.currentVideoId = source;
+      if (this.state.adWasPaused && this.state.currentAdsInfo.currentAdItem.ssai) {
```

Comment:
@ipadilla4 do we need a null check here for `this.state.currentAdsInfo` and `this.state.currentAdsInfo.currentAdItem`? If not disregard